asdfghjkl/core.py

```diff
@@ -16,9 +16,6 @@
 def extend(model, *op_names, ignore_modules=None, map_rule=None, vectors: ParamVector = None) -> OperationContext:
     handles = []
     cxt = OperationContext(vectors=vectors)
-    has_fwd_op = any(op_name in FWD_OPS for op_name in op_names)
-    has_bwd_op = any(op_name in BWD_OPS for op_name in op_names)
-    has_bwd_op_with_inputs = any(op_name in BWD_OPS_WITH_INPUTS for op_name in op_names)
```

Comment:
Sometimes, op_names = ['kron'] which is not in FWD_OPS.
We can change FWD_OPS, or check it after parsing 'kron'

Reply:
OP_COV_KRON is in FWD_OPS
https://github.com/kazukiosawa/asdfghjkl/blob/0.1/asdfghjkl/operations/operation.py#L34

Reply:
They are not the same, sometimes, op_names=[SHAPE_KRON]